### DISCUSSION TITLE
Any interest or issues with DynamoDB as the backing datastore?

### DISCUSSION THREAD

**Discussion by byamagata** [authzed/spicedb#1802] [Q&A] [answered]
I've been looking into spicedb for a while now, but really want to use dynamodb as the backing datastore.  Is this something that has been discussed?  Is there any desire for this?  Or any reasons why this couldnt be implemented?  

I dont mind doing it myself, just want to know if there are any issues with dynamodb as the database before jumping in.

**Comment by josephschorr** [comment, marked as answer]
See https://github.com/authzed/spicedb/issues/1083 for the discussion

No issues besides a question around how full consistency operations will be handled, but that likely would require the use of non-global tables
